### ISSUE TITLE
Installed package won't load in HoloLens 2

### ISSUE THREAD

**Issue by sPoss678** [MicrosoftDocs/mixed-reality#2197]
I've gone through this tutorial twice but both results are the same. The Unreal package builds successfully and the appxbundle installs on my HL2 but every time I start it, I see the loading mesh and hear the app launch sound then the app immediately closes back to only show the app launch window.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 41c8f2c0-77cc-9ae9-562d-16d163dff85f
* Version Independent ID: 3e0c6539-52e5-a851-7f2f-03fa2f7b3789
* Content: [6. Packaging & deploying to device or emulator - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/unreal-uxt-ch6)
* Content Source: [mixed-reality-docs/unreal-uxt-ch6.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/unreal-uxt-ch6.md)
* Product: **mixed-reality**
* Technology: **mr-dev**
* GitHub Login: @sw5813
* Microsoft Alias: **suwu**

**Comment by sw5813** [comment]
Are you able to get any other apps built with Unreal, deployed to, and running on device? What about with engines besides Unreal, like Unity? And have you confirmed that you're on the latest version of the OS?

**Comment by sPoss678** [comment]
This tutorial is my first attempt with Unreal but I have worked with Unity for a few years now and have no problem deploying working apps in that workflow.

HoloLens is running 10.0.18362.1042 and the project is targeting the 10.0.18362.0.
[cid:image001.png@01D627AF.E6947F60]

Additionally, both the HoloLens and Microsoft Windows Mixed Reality Plugins are enabled and running; and the editor and pc have been restarted multiple times.
[cid:image003.png@01D627B0.BDC044B0]

[cid:image007.png@01D627B0.BDC044B0]

[cid:image008.png@01D627B0.BDC044B0]

I have factory reset the HoloLens as well and redeployed but I get the same results.

**Comment by sPoss678** [comment]
Solved. When I created the plugins folder and copied in the UXTools I must have created a second copy. One at .../Plugins/UXTools and another at .../Plugins/UXTools/UXTools/. In the UE build log, it states that 2 copies were found and that the second would be ignored but this seems to produce the problem I was seeing. After removing the second copy and restarting UE, a new build and deploy works as expected.
